static/scss/checkout.scss

```diff
@@ -1,18 +1,170 @@
 .checkout-page {
-  margin: 100px;
+  @include media-breakpoint-up (lg) {
```

Comment:
one general comment here, there are several components with `${SOMETHING}-row` for a class that then have `display: flex; flex-direction: row` on them - it might make the CSS a little more general and save some code to have a separate `.row` class which applied those styles and then have only the styles specific to each class differing between them.

Reply:
Sounds good